spec/sunomono_test.rb

```diff
@@ -0,0 +1,366 @@
+        expect(Dir.entries("#{@project_name}/features/ios/screens")).
+            to include("#{feature_name}_screen.rb")
+      end
+
+      it 'Generate feature in pt' do
+        system "suno new '#{@project_name}'", :out => File::NULL
+
+        feature_name = 'sunomono'
+
+        Dir.chdir(@project_name)
+
+        system "suno generate feature '#{feature_name}' --lang=pt", :out => File::NULL
+
+        Dir.chdir('../')
+
+        expect(File.readlines("#{@project_name}/features/#{feature_name}.feature")).
+            to include("# language: pt\n", "Funcionalidade: #{feature_name.capitalize} \n", "\n", "  Contexto:\n", "    # Insira os passos\n", "    \n", "  Cenário: Primeiro Cenario\n", "    # Insira os passos\n")
+        expect(File.readlines("#{@project_name}/features/step_definitions/#{feature_name}_steps.rb")).
+            to include("######### DADO #########\n", "\n", "######### QUANDO #########\n", "\n", "######### ENTãO #########")
+      end
+
+      it 'Generate feature in en' do
+        system "suno new '#{@project_name}'", :out => File::NULL
+
+        feature_name = 'sunomono'
+
+        Dir.chdir(@project_name)
+
+        system "suno generate feature '#{feature_name}'", :out => File::NULL
+
+        Dir.chdir('../')
+
+        expect(File.readlines("#{@project_name}/features/#{feature_name}.feature")).
+            to include("# language: en\n", "Feature: #{feature_name.capitalize} \n", "\n", "  Background:\n", "    # Insert steps\n", "    \n", "  Scenario: First Scenario\n", "    # Insert steps\n")
+        expect(File.readlines("#{@project_name}/features/step_definitions/#{feature_name}_steps.rb")).
+            to include("######### GIVEN #########\n", "\n", "######### WHEN #########\n", "\n", "######### THEN #########")
+      end
+
+      it 'Generates with alias command g' do
+        system "suno new '#{@project_name}'", :out => File::NULL
+
+        Dir.chdir(@project_name)
+
+        feature_name = 'rspec_sunomono'
+
+        system "suno g feature '#{feature_name}'", :out => File::NULL
+
+        Dir.chdir('../')
+
+        expect(Dir.entries("#{@project_name}/features")).
+            to include("#{feature_name}.feature")
+        expect(Dir.entries("#{@project_name}/features/step_definitions")).
+            to include("#{feature_name}_steps.rb")
+        expect(Dir.entries("#{@project_name}/features/android/screens")).
+            to include("#{feature_name}_screen.rb")
+        expect(Dir.entries("#{@project_name}/features/ios/screens")).
+            to include("#{feature_name}_screen.rb")
+      end
+    end
+
+    context 'Try Generates a feature with a invalid argument' do
+      it 'Feature will be not created' do
+        system "sunomono new '#{@project_name}'", :out => File::NULL
+
+        feature_name = 'rspec_sunomono'
+
+        Dir.chdir(@project_name)
+
+        system "sunomono generate feature '#{feature_name}' invalid_argument", :out => File::NULL
```

Comment:
I am not talking about the role message, but to check if there was an "Error" word in the console output, exactly the same way you do with `sunomono --version` test